### PR TITLE
Correct typos in PA documentation

### PR DESCRIPTION
**Pull request by luispinedamx**
Correcting several typos across the documentation:

- Common typo of a space before a colon " :" which is correct in French, but not in English.
- Capitalization of **SSH** and **URL**.
- Correct spelling of Pro**A**ctive.

Note that obviously there are contexts where these corrections don't apply, e.g.:

`exists  kpi : NodeMetrics( url == "#nodeUrlVar" )`

Feel free to reach to me if you know other common mistakes that we can correct for the release.

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -1757,7 +1757,7 @@ NOTE: On Mac OS X, the default temporary folder is not shared between all users.
 To use a shared temporary folder, you need to set the _$TMPDIR_ environment variable and the Java property _java.io.tmpdir_
 to /tmp before starting the ProActive Node.
 
-NOTE: the same applies on Windows. To use a shared temporary folder, create a folder shared by all proactive users e.g. C:\TEMP and define
+NOTE: the same applies on Windows. To use a shared temporary folder, create a folder shared by allProActive users e.g. C:\TEMP and define
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: there is a typo here

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -1789,15 +1789,15 @@ proactive-node -Dpas.launcher.forkas.method=key
 
 This configuration, only availabe on linux nodes, allows the impersonation to be performed using passwordless sudo.
 
-To enable it at the system level, edit the /etc/sudoers file to allow passwordless sudo from the account running the proactive node
+To enable it at the system level, edit the /etc/sudoers file to allow passwordless sudo from the account running theProActive node
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: same

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -1789,15 +1789,15 @@ proactive-node -Dpas.launcher.forkas.method=key
...
 to any users which require impersonation. Passwordless sudo should be enabled for any command.
 
 For example, the following line will allow the proactive account to impersonate to any user:
 ----
 proactive ALL=(ALL) NOPASSWD: ALL
 ----
 
-To enable this configuration on the proactive node, start it with the system property `pas.launcher.forkas.method` to _none_
+To enable this configuration on theProActive node, start it with the system property `pas.launcher.forkas.method` to _none_
```
Comment (fviale) [inline src/docs/admin/ProActiveAdminGuide.adoc]: same

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 4]
```diff
@@ -228,7 +228,7 @@ However, it implements a different instance management strategy that reduces the
 
 The configuration of the AWS Autoscaling infrastructure is subjected to several requirements.
 
-	1. The administrator needs both an *AWS access key* and an *AWS secret access key* to enable proactive to authenticate against AWS.
+	1. The administrator needs both an *AWS access key* and an *AWS secret access key* to enableProActive to authenticate against AWS.
```
Comment (fviale) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: typo

File: src/docs/user/ProActiveUserGuide.adoc [hunk 5]
```diff
@@ -1551,8 +1551,8 @@ Here is the list of topologies we support:
 === Handling failures
 
 It’s possible to have errors when your Job is executed. *ProActive Scheduler* provides several mechanisms to deal with exceptional situations in your Jobs.
-If a Task contains errors during Job execution, the *Proactive Scheduler* automatically restarts this Task. The number of automatic restarts is defined by the Workflow owner by assigning an integer value to the counter *Number of Automatic Restarts*.
-The user also defines *Where he wants that the scheduler restarts In-Error tasks*, i.e., on the same Proactive node or on another Proactive node.
+If a Task contains errors during Job execution, the *ProActive Scheduler* automatically restarts this Task. The number of automatic restarts is defined by the Workflow owner by assigning an integer value to the counter *Number of Automatic Restarts*.
+The user also defines *Where he wants that the scheduler restarts In-Error tasks*, i.e., on the same ProActive node or on another ProActive node.
```
Comment (fviale) [inline src/docs/user/ProActiveUserGuide.adoc]: you can use Node with a capital N here

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (luispinedamx) [reply src/docs/admin/ProActiveAdminGuide.adoc]: damn, I must have deleted a space :( thanks for noticing

**Comment by luispinedamx** [comment]
Thanks @fviale, fixed the missing space.

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 6]
```diff
@@ -634,15 +634,15 @@ It is possible to enrich the Machine Learning Bucket by adding your own tasks. (
 
 It is also possible to customize the code of the generic Machine Learning tasks. In this case, you need to drag and drop the targeted task to modify its code in the *Task Implementation* section.
 
-NOTE: It is also possible to add or/and delete variables of each task, set your own fork environments etc. More details available on link:../user/ProActiveUserGuide.html#_studio_tasks[Proactive User Guide^]
+NOTE: It is also possible to add or/and delete variables of each task, set your own fork environments etc. More details available on link:../user/ProActiveUserGuide.html#_studio_tasks[ProActive User Guide^]
```
Comment (tao-xinxiu) [inline src/docs/MLOS/MLOSUserGuide.adoc]: ... set your own fork environments **,** (add comma) etc.

File: src/docs/PCW/PCWUserGuide.adoc [hunk 7]
```diff
@@ -224,7 +224,7 @@ Each monitoring type has an implementation of a Polling class. The polling param
     </kpis>
 -----------------
 The code above describes a polling configuration for Monitoring physical metrics. This  particular example is for monitoring CPUs. +
-The polling and calm periods correspond to the frequency of collecting the monitoring metrics. The concrete set of parameters rely on the implementation of a Polling class. In our example, the class is *Jmx_Metrics*. This implementation takes two parameters: the jmx node url and the jmx credentials.
+The polling and calm periods correspond to the frequency of collecting the monitoring metrics. The concrete set of parameters rely on the implementation of a Polling class. In our example, the class is *Jmx_Metrics*. This implementation takes two parameters: the JMX node URL and the JMX credentials.
 The actual metric to monitor is represented by the *_<kpi>_* tag. Several metrics can be monitored at the same time. For our example after variables replacement the monitoring metric will be _"sigar:Type=CpuUsage Combined"._
```
Comment (tao-xinxiu) [inline src/docs/PCW/PCWUserGuide.adoc]: The concrete set of parameters rel**ies** on

File: src/docs/admin/AdminTutorials.adoc [hunk 8]
```diff
@@ -20,22 +20,22 @@ The following steps configure the Node with a sample user called Gord.
...
 . Add a *Run As Me* user, which needs to be part of the activeeon group _useradd -g activeeon -m Gord_
-. Create an ssh key for the *Run As Me* user (Gord) _ssh-keygen -t rsa_
+. Create an SSH key for the *Run As Me* user (Gord) _ssh-keygen -t rsa_
 . Go into the _/home/Gord/.ssh_ directory and copy Gord’s public key to the authorized keys _cat id_rsa.pub >> authorized_keys_
-. Log into the activeeon account
+. Log into the _activeeon_ account
 . Start the ProActive Node with the property `-Dpas.launcher.forkas.method=key`
 
-NOTE: The proactive user got a password set with _passwd proactive_ to execute the sudo command. Whereas the run as me user (Gord) got a SSH key set. If you wish you can assign a password to Gord _passwd Gord_.
+NOTE: The proactive user got a password set with _passwd proactive_ to execute the sudo command. Whereas the run as me user (Gord) got an SSH key set. If you wish you can assign a password to Gord _passwd Gord_.
```
Comment (tao-xinxiu) [inline src/docs/admin/AdminTutorials.adoc]: If you wish, (add comma) you can assign

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 9]
```diff
@@ -517,7 +517,7 @@ The main troubles you may have to face are the following ones:
...
-* The node seems to be well started but you cannot see it in the Resource Manager interface : in this case, make sure that the *port number* is the good one. Do not forget that the runtime port number is incremented from the initial ProActive Resource Manager port number. You can see exactly on which port your runtime has been started looking at the log file described above.
+* The node seems to be well started but you cannot see it in the Resource Manager interface: in this case, make sure that the *port number* is the good one. Do not forget that the runtime port number is incremented from the initial ProActive Resource Manager port number. You can see exactly on which port your runtime has been started looking at the log file described above.
```
Comment (tao-xinxiu) [inline src/docs/admin/ProActiveAdminGuide.adoc]: The node seems to be well started, (add comma) but you cannot

File: src/docs/PCW/PCWUserGuide.adoc (hunk 7 above)
Comment (luispinedamx) [reply src/docs/PCW/PCWUserGuide.adoc]: good catch!

**Comment by luispinedamx** [comment]
Thanks @tao-xinxiu, changes applied

**Pull request merged**